wd_capabilities.au3

```diff
@@ -179,75 +176,75 @@ Func _WD_CapabilitiesAdd($key, $value1 = Default, $value2 = Default)
 	Local Const $s_Parameters_Info = '     $key = ' & $key & '     $value1 = ' & $value1 & '     $value2 = ' & $value2
 
 	If StringRegExp($key, $_WD_KEYS__MATCHTYPES, $STR_REGEXPMATCH) Then ; check if alwaysMatch|firstMatch
-		Local $iResult = __WD_CapabilitiesInitialize($key, $value1)
-		__WD_ConsoleWrite($sFuncName & ": TESTING #" & @ScriptLineNumber & $s_Parameters_Info & "  :: DEBUG", $_WD_DEBUG_Full)
-		If Not @error Then $_WD_CAPS__CURRENTIDX = $iResult
-		Return SetError(@error, @extended, $_WD_CAPS__CURRENTIDX)
+		__WD_CapabilitiesNotation($key, $value1)
+		__WD_ConsoleWrite($sFuncName & ': #' & @ScriptLineNumber & $s_Parameters_Info, $_WD_DEBUG_Full)
+		Return SetError(@error, @extended)
```

Comment:
Shouldn't this be calling __WD_Error here?

Reply:
Take a look on: 
[__WD_CapabilitiesNotation() + __WD_Error()](https://github.com/Danp2/au3WebDriver/pull/285/commits/4be5d8ddc00989389136f83f135846bec2665a1e)

Reply:
Why would you choose to do it this way? IMO, you are just going to confuse anyone looking at the logs because they called _WD_CapabilitiesAdd, but the logs will show __WD_CapabilitiesNotation. 

Technically, you no longer need to breakout the code in __WD_CapabilitiesNotation since it is only called once by _WD_CapabilitiesAdd.

Reply:
I was not aware of that.
I need to think about.

Reply:
Will it be fine if I back to _WD_CapabilitiesInitialization ?

Reply:
Take a look on:
[__WD_CapabilitiesNotation removed](https://github.com/Danp2/au3WebDriver/pull/285/commits/265f1f0e0e664324bfd34951557bc74a761678cc)

Reply:
My initial concern was with the omission of calling __WD_Error. Looking at the most recent changes, I don't see how you have addressed this concern.

Reply:
do you refer to this __WD_Error()
![image](https://user-images.githubusercontent.com/11089482/163468631-5bce15a6-512b-4a6c-93a2-036985015246.png)
They are needed there.


This commit: [__WD_CapabilitiesNotation removed](https://github.com/Danp2/au3WebDriver/pull/285/commits/265f1f0e0e664324bfd34951557bc74a761678cc)

Was related to your statement:

> Technically, you no longer need to breakout the code in __WD_CapabilitiesNotation since it is only called once by _WD_CapabilitiesAdd.

It is NOT obvious that you call it once.
For example: If you want to have `alwaysMatch` + 3*`firstMatch` then you need to call this 4 times.

I only removed `__WD_CapabilitiesNotation` to fit to your quite a reasonable statement:

> you are just going to confuse anyone looking at the logs because they called _WD_CapabilitiesAdd, but the logs will show __WD_CapabilitiesNotation.

Reply:
You misunderstand. I'm not asking you to remove calls to __WD_Error. I said "Shouldn't this be calling __WD_Error here?" and "omission of calling __WD_Error", meaning that there are places in wd_capabilities where you return from a function WITHOUT calling __WD_Error, which I was questioning if that needed to be corrected.

Reply:
ah... lets see..

Reply:
[Successfully used $s_MatchType](https://github.com/Danp2/au3WebDriver/pull/285/commits/fb64659317e1e64db89aa398f22823ad633af8e8)

Reply:
[Capability.....Suplemented into....](https://github.com/Danp2/au3WebDriver/pull/285/commits/b32d7b10a77370a625eb5e0c513bf70b636fd797)

Reply:
[_WD_CapabilitiesAdd/_WD_CapabilitiesDefine success](https://github.com/Danp2/au3WebDriver/pull/285/commits/49cdb8cddd170585b88d8ac31c4bd2518ce19838)

Reply:
> meaning that there are places in wd_capabilities where you return from a function WITHOUT calling __WD_Error, which I was questioning if that needed to be corrected.

when it comes about: `_WD_CapabilitiesGet()` I think that this function should not use `__WD_Error`